doc/admin-guide/files/remap.config.en.rst

```diff
@@ -675,18 +764,18 @@ PURGE  \-             \-             \-             deny             denied  (40
 PUSH   \-             \-             \-             deny             denied  (403)
 ====== ============== ============== ============== ================ =============
 
-With the "Match on IP only Policy", the in-line filter works like an :file:`ip_allow.yaml` rule applies to all requests to
-``www.example.com`` that denies ``POST`` requests and implicitly allows all other methods:
+With the "Modern" policy, the in-line filter works like an :file:`ip_allow.yaml` rulei: it applies to all requests to
+``www.example.com`` and denies ``POST`` requests while allowing all other methods:
 
 ====== ================ ============== ============== ============= =============
 Method In-line Filter   Named Filter 1 Named Filter 2 ip_allow.yaml result
 ====== ================ ============== ============== ============= =============
-GET    allow (implicit) \-             \-             \-            allowed (200)
+GET    allow            \-             \-             would allow   allowed (200)
```

Comment:
What do you mean by `would allow` and `would deny` in this table?

Reply:
With the Match on IP Only policy, only the inline ACL rule takes effect. None of the named ACL filters nor the ip_allow rule is queried. So I thought it helpful to make it clear in the table that the other rules are not actually denying or allowing something, rather they "would allow" if they were in effect. Note before, we had some `allow` and `deny` entries while those filters were not in effect. So I wanted to indicate that _had_ they run they would make that impact, thus "would deny" or "would allow". 

I suppose we should do one of two things:

* Change all of the other entries to `\-` other than the in-line filter column and the result column.
* Fill in all the other entries with "would allow/deny" because, if they were in effect, with Match on IP Only policy, they would completely specify the behavior.

I guess my vote would be to change all to `\-`.